packages/chakra-components/src/theme/vote.ts

```diff
@@ -0,0 +1,27 @@
+import { createMultiStyleConfigHelpers } from '@chakra-ui/styled-system'
+
+export const VotwWeightAnatomy = [
+  // confirmation wrapper box
+  'wrapper',
+  // title text
+  'weight',
+]
+
+const { defineMultiStyleConfig: defineVoteWeightTipStyle, definePartsStyle: defineVoteWeightParts } =
+  createMultiStyleConfigHelpers(VotwWeightAnatomy)
+
+export const VoteWeightTheme = defineVoteWeightTipStyle({
+  baseStyle: defineVoteWeightParts({
+    wrapper: {
+      mt: 4,
+      w: 'full',
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+      gap: 2,
+    },
+    weight: {
+      fontWeight: 'bold',
+    },
```

Comment:
Way too much styling for the base component IMHO

Reply:
Is not to much, just for avoid to style on the ui-scaffold.

Reply:
This bf4c190 is simplier and doesn't breaks nothing on ui scaffold. I just added a margin bottom there: [f71f4b6](https://github.com/vocdoni/ui-scaffold/pull/669/commits/f71f4b6)